From 8902f4d7b7c532592d1a34ad117698d3e380e9e1 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Fri, 10 Mar 2017 22:23:06 +0100
Subject: [PATCH] eglfs: fix x11 header related compile failure

Add egl config and QT_EGL_NO_X11 define (as all other eglfs project
files do).

Task-number: QTBUG-59427
Change-Id: Ifbb11eae0fdf0e58c0b7feecb9a7914a889c8f77

Upstream: https://codereview.qt-project.org/188158
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 src/plugins/platforms/eglfs/eglfs-plugin.pro | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/plugins/platforms/eglfs/eglfs-plugin.pro b/src/plugins/platforms/eglfs/eglfs-plugin.pro
index cf4863975a..ec229796e5 100644
--- a/src/plugins/platforms/eglfs/eglfs-plugin.pro
+++ b/src/plugins/platforms/eglfs/eglfs-plugin.pro
@@ -2,6 +2,11 @@ TARGET = qeglfs
 
 QT += eglfsdeviceintegration-private
 
+CONFIG += egl
+
+# Avoid X11 header collision, use generic EGL native types
+DEFINES += QT_EGL_NO_X11
+
 SOURCES += $$PWD/qeglfsmain.cpp
 
 OTHER_FILES += $$PWD/eglfs.json
-- 
2.11.0

